cocos/ui/UIEditBox/UIEditBoxImpl-common.cpp

```diff
@@ -124,7 +124,7 @@ void EditBoxImplCommon::setInactiveText(const char* pText)
     
 void EditBoxImplCommon::setFont(const char* pFontName, int fontSize)
 {
-    this->setNativeFont(pFontName, fontSize);
+    this->setNativeFont(pFontName, fontSize * _label->nodeToWorldTransform().a);
```

Comment:
I do sync the transform info to native control in the `draw` method:

```
    if(flags)
    {
        auto rect = ui::Helper::convertBoundingBoxToScreen(_editBox);
        this->updateNativeFrame(rect);
    }
```

Could you add a test for the scaling issue?